spec/tasks/gen/model_spec.cr

```diff
@@ -50,14 +50,43 @@ describe Gen::Model do
     end
   end
 
+  it "generates a model with nilable columns" do
```

Comment:
Rather than adding a new block, what do you think about passing a nilable column in the spec above? That way there aren't so many shared setup and expectation lines

Reply:
Can do, @paulcsmith ! I'll resubmit this week